_studio/shared/umc/codec/av1_dec/src/umc_av1_utils.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2012-2020 Intel Corporation
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#include "umc_defs.h"
+#ifdef MFX_ENABLE_AV1_VIDEO_DECODE
+
+#include "umc_structures.h"
+#include "umc_vp9_utils.h"
+#include "umc_av1_utils.h"
+
+namespace UMC_AV1_DECODER
+{
+    void SetSegData(SegmentationParams & seg, uint8_t segmentId, SEG_LVL_FEATURES featureId, int32_t seg_data)
+    {
+        VM_ASSERT(seg_data <= SEG_FEATURE_DATA_MAX[featureId]);
+        if (seg_data < 0)
+        {
+            VM_ASSERT(SEG_FEATURE_DATA_SIGNED[featureId]);
+            VM_ASSERT(-seg_data <= SEG_FEATURE_DATA_MAX[featureId]);
+        }
+
+        seg.FeatureData[segmentId][featureId] = seg_data;
+    }
+
+    void SetupPastIndependence(FrameHeader & info)
+    {
+        ClearAllSegFeatures(info.segmentation_params);
+
+        SetDefaultLFParams(info.loop_filter_params);
+    }
+
+    inline uint32_t av1_get_qindex(FrameHeader const& fh, uint8_t segmentId)
```

Comment:
style fix: the most functions have pascal notation but this - snake one . Which style we use? Is there logic to use different ones?

Reply:
I will change it to pascal notation.